rfc/rfc-0003-developer-utils.md

```diff
@@ -0,0 +1,74 @@
+# RFC-0003: A pod with utilities for better developer experience
+
+**Authors:** Swapnil Bawaskar
+
+**Status:**
+
+**Pull Request URL:** [#1362](https://github.com/projectriff/riff/pull/1362)
+
+**Superseded by:** N/A
+
+**Supersedes:** [RFC 0001](https://github.com/projectriff/riff/pull/1359)
+
+**Related:**
+
+
+## Problem
+A developer would like to invoke the function for which they just created the `core deployer` or the `knative deployer`.
+When the streaming runtime is installed, they would like to test the processor by getting events into the stream and look into the contents of the stream. There is no one step mechanism for accomplishing any of these currently.
+An [earlier proposal](https://github.com/projectriff/riff/pull/1359) was to provide some of this functionality in the riff cli, this proposal expands on [Mark's comment](https://github.com/projectriff/riff/pull/1359#discussion_r348617981).
+
+### Anti-Goals
+We will only address this problem for development/demos, not production, so topics like auth/authz are out of scope for this document.
+
+## Solution
+We will ask the developers to run a `riff-developer-utils` pod in their development cluster. This pod will bundle a few binaries/scripts that users can invoke using `kubectl exec`. Since the commands will be run in-cluster, users won't need to run `kubectl port-forward`.
+We will have the following commands to start with:
```

Comment:
the way to avoid the collision would be to avoid doing the host name lookup in the dev-utils pod. Instead pass in a hostname. But at that point, you might as well just call `curl` directly.

Reply:
dropped the invoke commands in favor of curl